code/workspaces/web-app/src/components/assetRepo/AddRepoMetadataDetails.js

```diff
@@ -121,7 +121,7 @@ export function AddRepoMetadata({ handleSubmit, onCancel }) {
   };
 
   const typeOptions = [
```

Comment:
Should this be constructed from the config map?

Reply:
Yes, but it makes sense to wait until after https://jira.ceh.ac.uk/browse/NERCDL-841 "Update web-app to use information from config-maps in a consistent manner" is done.  I've added it as a comment to that story.